spec/resources/mod_auth_cas_spec.rb

```diff
@@ -23,6 +23,7 @@
           cache_dir: '/var/cache/apache2',
           login_url: 'https://login.example.org/cas/login',
           validate_url: 'https://login.example.org/cas/serviceValidate',
+          directives: nil,
```

Comment:
An example set of directives would be really helpful here to work this code out.

Reply:
@damacus How does the latest addition look?